doc/crm.8.adoc

```diff
@@ -2104,6 +2104,71 @@ Example:
 utilization xen1 set memory 4096
 ...............
 
+[[cmdhelp.sbd,SBD management]]
+=== `sbd` - SBD management
+
+This level is for managing the SBD (STONITH Block Device) daemon.
+
+[[cmdhelp.sbd.configure,configure SBD]]
+==== `configure`
+
+Configure the SBD daemon for both disk-based and disk-less mode.
+
+Main functionailities include:
+- Show configured disk metadata
+- Show contents of /etc/sysconfig/sbd
+- Show SBD related cluster properties
+- Newly setup SBD configuration on a running cluster
+- Update the existing parameters
+- Add more devices to the existing disk-based SBD configuration
+
+For more details on SBD and related parameters, please see man sbd(8).
+
+Usage:
+...............
+# For disk-based SBD
+crm sbd configure show [disk_metadata|sysconfig|property]
+crm sbd configure [device=<dev>]... [watchdog-device=<dev>] [watchdog-timeout=<integer>] [allocate-timeout=<integer>] [loop-timeout=<integer>] [msgwait-timeout=<integer>]
```

Comment:
Some description to set up the proper user expectation. Something like,

```suggestion
crm sbd configure [device=<dev>]... [watchdog-device=<dev>] [watchdog-timeout=<integer>] [allocate-timeout=<integer>] [loop-timeout=<integer>] [msgwait-timeout=<integer>]
        NOTE: 
        1. for the new device, it copies the disk metadata from the first existing SBD disk device.
        2. for the existing "device=<dev>", it will skip disk metadata operation and if no change.                       
```

Reply:
Why bother to write such complex document and expect users to read them? This can be avoided by better UI design. For example, we can reject `configure device=...` when a disk exists and require using `configure device+=...` to add a new disk.

Reply:
> Why bother to write such complex document and expect users to read them? This can be avoided by better UI design. For example, we can reject `configure device=...` when a disk exists and require using `configure device+=...` to add a new disk.

It all about setting up the proper user expectation. I would try to rewrite a bit. Or, we have the better place or the better UI to deliver such expecation.